client.go

```diff
@@ -8,9 +8,9 @@ import (
 	"io/ioutil"
 	"os"
 	"path/filepath"
+	"runtime"
```

Comment:
`goimports` is run on save, which (among other things) alphabetically sorts imported packages